module/TaskManagement/Module.php

```diff
@@ -35,7 +35,9 @@ public function getControllerConfig()
 					$taskService = $locator->get('TaskManagement\TaskService');
 					$streamService = $locator->get('TaskManagement\StreamService');
 					$acl = $locator->get('Application\Service\Acl');
+					$assignmentOfSharesConfig = $locator->get('Config')['assignment_of_shares'];
 					$controller = new TasksController($taskService, $streamService, $acl);
+					$controller->setIntervalForCloseTasks($assignmentOfSharesConfig['TaskManagement\TimeboxForAssignmentOfShares']);

```

Comment:
Nessuna utilità specifica. Racchiudo allora questa informazione in un'unica chiave 'assignment_of_shares_timebox'